schema/schema.go

```diff
@@ -110,3 +110,50 @@ func (s *WebRPCSchema) GetServiceByName(name string) *Service {
 	}
 	return nil
 }
+
+func MatchServices(s *WebRPCSchema, services map[string]struct{}) *WebRPCSchema {
```

Comment:
Should this be a method on `*WebRPCSchema`?

why `map[string]struct{}`?

```suggestion
// Filter services, remove those not provided in the slice.
func (s *WebRPCSchema) FilterServices(services []string) {
```

Reply:
I did not want to expose these functions to `templates`.  See: https://github.com/webrpc/webrpc/blob/ca6fbe4f0e3e8948c832bc427a4ea2aacd172b44/gen/gen.go#L42

map[string] it's easier to look for specific service or annotation in map then slice :man_shrugging: